Code/Source/HumanWorker/NavigationMeshOrchestratorComponent.h

```diff
@@ -51,5 +51,7 @@ namespace ROS2::HumanWorker
         float m_updateFrequency{ 0.1f }; //!< In Hertz.
         float m_elapsedTime{ 0.0f }; //!< In seconds.
         bool m_initialUpdate{ false };
+        int m_delayedTickUpdate{ 0 };
+        bool m_delayedTickUpdateActive{ false };
```

Comment:
The reason for the delayed tick update is that there was a case of objects being spawned in the first tick of the simulation. This would require a notification system for the mesh creation (the spawned objects would send the notification). Usually setting this value to 1 would solve this problem.